### PR TITLE
Add image columns

### PR DESCRIPTION
**Pull request by pxlrbt**
Adds image colums for products and posts:

Bildschirmfoto 2022-03-22 um 14 10 44

Not sure how the demo app is deployed. Probably you need to clear the storage folder on every deploy/reset.

### CODE REVIEW

File: app/Filament/Resources/Shop/ProductResource.php [hunk 1]
```diff
@@ -213,6 +213,10 @@ public static function getFormSchema(string $layout = Forms\Components\Grid::cla
     public static function getTableColumns(): array
     {
         return [
+            Tables\Columns\ImageColumn::make('media')
```
Comment (danharrin) [inline app/Filament/Resources/Shop/ProductResource.php]: We have a SpatieMediaLibraryImageColumn right?

**Review by danharrin — CHANGES_REQUESTED** [review]

File: app/Filament/Resources/Shop/ProductResource.php (hunk 1 above)
Comment (pxlrbt) [reply app/Filament/Resources/Shop/ProductResource.php]: Thanks. Didn't see that one.

File: database/seeders/DatabaseSeeder.php [hunk 2]
```diff
@@ -15,11 +15,15 @@
...
 use App\Models\User;
 use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\Storage;
 
 class DatabaseSeeder extends Seeder
 {
     public function run(): void
     {
+        // Clear images
+        Storage::deleteDirectory('public');
```
Comment (pxlrbt) [inline database/seeders/DatabaseSeeder.php:25]: @danharrin Is this okay to clean up existing images before seeding? Or do you prefer something else?

Comment (danharrin) [reply database/seeders/DatabaseSeeder.php:25]: fine by me!

**Pull request merged**